tests/lib/bash/test_header

```diff
@@ -332,6 +339,19 @@ grep_ok() {
     fail "${TEST_NAME}"
 }
 
+grep_fail() {
```

Comment:
Ha, I added `grep_fail()` in #2868 as well. Same number of lines, but with the logic reversed.

Reply:
It's a useful one, I've got a strange feeling that I've written this one more than once.

Reply:
Yeah, I was surprised it didn't exist (thought it did).